packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -738,6 +738,10 @@ class AndroidWebResourceError extends WebResourceError {
         );
 
   /// Gets the URL for which the failing resource request was made.
+  @override
+  // Temporarily ignored as this will be added to the parent platform interface
+  // class.
+  // ignore: overridden_fields
```

Comment:
This will be included in the platform interface PR to keep CI happy. This shouldn't be a breaking change.